java/java.lsp.server/src/org/netbeans/modules/java/lsp/server/explorer/TreeItem.java

```diff
@@ -42,6 +42,7 @@
     // rendered less prominent. When true, it is derived from resourceUri
     public String description;
     // ?: string | Uri | {dark: string | Uri, light: string | Uri} | ThemeIcon
+    public IconDescriptor iconDescriptor;
```

Comment:
Is the comment on line 45 supposed to apply to `IconDescriptor`? It doesn't seem to match.

Reply:
Fixed in b20f0d6